src/screens/User/User.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import { View, Image, TouchableOpacity, Linking, FlatList } from 'react-native'
+import { Heading, HeadingType, PlainText } from 'src/components'
+import { Podcast, User } from 'src/common/types'
+import PodcastItem from './components'
+import { MockedUser, MockedPodcasts } from './mockedData'
+import Check from 'src/assets/images/checkMark.svg'
+import AtMark from 'src/assets/images/atMark.svg'
+import LogOut from 'src/assets/images/iconmonstr-log-out-16.svg'
+import styles from './styles'
+
+type Props = {
+  podcasts: Podcast[]
+  user: User
+}
+
+const userPageInfo: Props = {
+  podcasts: MockedPodcasts,
+  user: MockedUser
+}
+
+const UserPage: React.FC<Props> = () => {
+  const handleLogOut = () => {
+    return 0
+  }
+
+  const renderItem = ({ item }: { item: Podcast }) => (
+    <PodcastItem podcast={item} />
+  )
+
+  return (
+    <View style={styles.container}>
+      <View>
+        <Heading type={HeadingType.LARGE} label="Profile" />
+      </View>
+
+      <View style={styles.userInfoContaienr}>
+        <View>
+          <Image
+            width={118}
+            height={118}
+            source={{ uri: userPageInfo.user.image?.url }}
+            style={styles.image}
+          />
+        </View>
+        <View style={styles.nameContainer}>
+          <Heading
+            type={HeadingType.MEDIUM}
+            label={`${userPageInfo.user.firstName} ${userPageInfo.user.lastName}`}
+          />
+          <View style={styles.userContacts}>
+            <View style={styles.userInfoItem}>
+              <Check width={15} />
+              <PlainText
+                style={styles.userInfoItemText}
+                label={userPageInfo.user.nickname}
+              />
+            </View>
+            <TouchableOpacity
+              activeOpacity={0.7}
+              style={styles.userInfoItem}
+              onPress={() =>
+                Linking.openURL(
+                  `mailto://${userPageInfo.user.email}}&subject=abcdefg&body=body`
+                )
+              }
+            >
+              <AtMark width={15} />
+              <PlainText
+                style={styles.userInfoItemText}
+                label={userPageInfo.user.email}
+              />
+            </TouchableOpacity>
+            <TouchableOpacity
+              activeOpacity={0.7}
+              style={styles.userInfoItem}
+              onPress={handleLogOut}
+            >
+              <LogOut width={15} />
+              <PlainText
+                style={styles.userInfoItemTextDelete}
+                label={'Log out'}
+              />
+            </TouchableOpacity>
+          </View>
+        </View>
+      </View>
+
+      <View style={styles.userDataContainer}>
+        <Heading type={HeadingType.MEDIUM} label="Podcasts" />
+        <FlatList
+          data={userPageInfo.podcasts}
+          renderItem={renderItem}
+          keyExtractor={(item) => item.name}
```

Comment:
Are you sure that `name` is good unique key?

Reply:
I cant add a number value, so thats all I can do, I guess